sklearn/datasets/descr/twenty_newsgroups.rst

```diff
@@ -62,7 +71,7 @@ attribute is the integer index of the category::
   >>> newsgroups_train.target.shape
   (11314,)
   >>> newsgroups_train.target[:10]
-  array([ 7,  4,  4,  1, 14, 16, 13,  3,  2,  4])
```

Comment:
What's happening here?

Reply:
I have no idea... I corrected it. 
(I may have had errors and thought that the dataset had been modified, but I've just tested it and it works as expected.)